lib/match.cpp

```diff
@@ -29,6 +28,10 @@ static int chkmask(const match_t &m, int maskbits) {
 }
 
 std::ostream &operator<<(std::ostream &out, match_t m) {
+    if (!m) {
+        out << "*";
+        return out;
+    }
```

Comment:
Could fold this into the final `else` just below (that case only runs when `m.word0|m.word1 == 0`, the same as `!m`)